sdks/python/apache_beam/io/gcp/bigquery.py

```diff
@@ -739,7 +739,7 @@ def estimate_size(self):
           self._job_name,
           self._source_uuid,
           bigquery_tools.BigQueryJobTypes.QUERY,
-          random.randint(0, 1000))
+          int(time.time()))
```

Comment:
Hmm, this assumes that no two users will start jobs from the same template at the same time, right ? I'm not sure if this is a strong enough assumption for public templates that might be used by a large number of users.

Reply:
added further randomization.